docs/apis/phi-silica.md

```diff
@@ -104,8 +104,8 @@ Phi Silica includes built-in text transformation capabilities (known as Text Int
 Supported skills include:
 
 - Text-to-table: Formats the prompt response into a structured table format, when appropriate.
-- Summarize: Returns a concise summary of the prompt text.
-- Rewrite: Rephrases the prompt text to optimize clarity, readability, and, when specified, tone (or style).
+- Summarize: Returns a concise summary of the prompt text, including support for "conversation" input such as an email thread.
+- Rewrite: Rephrases the prompt text to optimize clarity, readability, and, when specified, tone (or style). Tones include Professional, Friendly, Concise, and Custom [Experimental].
```

Comment:
@connorMicrosoft - These are not the values listed in https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.windows.ai.text.textrewritetone?view=windows-app-sdk-2.0-experimental.